corehq/toggles.py

```diff
@@ -298,15 +299,16 @@ def toggle_values_by_name(username=None, domain=None):
     'detail-list-tab-nodesets',
     'Associate a nodeset with a case detail tab',
     TAG_PRODUCT_PATH,
-    [NAMESPACE_DOMAIN],
     help_link='https://confluence.dimagi.com/display/internal/Case+Detail+Nodesets',
+    namespaces=[NAMESPACE_DOMAIN]
```

Comment:
@krsdimagi both versions of this are actually valid.  In python, you can use both "positional" arguments and "keyword" (aka "named") arguments.  The interplay between the two is a little strange, if you're interested in learning more, those are some terms to google.